src/main/java/com/varukha/onlinebookstore/exception/CustomGlobalExceptionHandler.java

```diff
@@ -47,11 +47,11 @@ protected ResponseEntity<Object> handleRegistrationException(
     }
 
     @ExceptionHandler(RuntimeException.class)
-    protected ResponseEntity<Object> handleAllErrors() {
+    protected ResponseEntity<Object> handleAllErrors(RuntimeException exception) {
         ErrorResponseDto response = new ErrorResponseDto(
                 LocalDateTime.now(),
                 HttpStatus.INTERNAL_SERVER_ERROR,
-                new String[]{ "Oops something wrong!"}
+                new String[]{ exception.getMessage()}
```

Comment:
As I told previously, not the best decision to return unknown server exception messages to the user